libs/features/mas/src/buildCheckoutUrl.js

```diff
@@ -155,7 +156,7 @@ export function buildCheckoutUrl(checkoutData) {
   const { env, items, workflowStep, ms, marketSegment, customerSegment, ot, offerType, pa, productArrangementCode, landscape, modal, ...rest } =
     checkoutData;
   const segmentationParameters = {
-    marketSegment: marketSegment ?? ms,
+    marketSegment: ms ?? marketSegment,
```

Comment:
Let's check if this change will not break anything for non-3in1 CTAs (the ones that have the modal=true param), if not, then all good 👍